cmd/bedpeOverlap/bedpeOverlap.go

```diff
@@ -262,7 +262,7 @@ func main() {
 		log.Fatalf("Error: overlapThreshold must be used with bedSelect")
 	}
 
-	if *overlapBoth == true && *bedSelect == false {
+	if *overlapBoth {
```

Comment:
```suggestion
	if *overlapBoth && !*bedSelect {
```

Reply:
Is this right? We don't want to fatal if overlapBoth is true and bedSelect is also true.

Reply:
Ah nice catch, I misinterpreted the cases, these are positional flags by the user